linkedin/linkedin.lua

```diff
@@ -0,0 +1,66 @@
+--[[
+# [LinkedIn API](https://developer.linkedin.com/docs/ref/v2/profile/basic-profile) connector.
```

Comment:
Link is pointing to non-existant page, should be this: https://developer.linkedin.com/docs/rest-api